PayPal/src/main/java/com/braintreepayments/api/paypal/AmountBreakdown.kt

```diff
@@ -0,0 +1,44 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * A recurring billing amount breakdown.
+ *
+ * This object is only used in [PayPalCheckoutRequest] to customize how the transaction amount is
+ * broken down. If provided, [itemTotal] becomes required. Some fields are conditionally required or
+ * not accepted based depending on the checkout flow (e.g., one-time vs subscription).
```

Comment:
Nit to clean up docstring:
```suggestion
 * This object can only used for the [PayPalCheckoutRequest] to customize how the transaction amount is
 * broken down. If [AmountBreakdown] is provided, [itemTotal] is required. Some fields are conditionally required or
 * not accepted depending on the checkout flow (e.g., one-time vs subscription).
```